sdks/java/maven-archetypes/examples-java8/generate-sources.sh

```diff
@@ -22,7 +22,7 @@
 # Usage: Invoke with no arguments from any working directory.
 
 # The directory of this script. Assumes root of the maven-archetypes module.
-HERE="$(dirname $0)"
+HERE="$( cd "$( dirname '$0' )" && pwd )"
```

Comment:
Tried `"$(dirname "$0")"` in local and [Jenkins](https://builds.apache.org/job/beam_PostCommit_Java_JDK_Versions_Test/13/jdk=JDK%201.8%20(latest),label=beam/) and build passed. Guess it's the prefer way to do it.